docs/integrations-guides/rollup-guides/integrations-overview.md

```diff
@@ -1,68 +1,78 @@
 engineers with a strong understanding of how an EigenDA integration would impact
-their tech stack and security model.
+their tech stack and security model. For full details, see the [EigenDA V2 integration spec](https://layr-labs.github.io/eigenda/integration.html#certblobtiming-validation).
 
-For any given rollup there are four main concerns inherent to an integration
+> Note: Each rollup stack uses slightly different terminology to refer to the same ideas.
+> We try to use the most general language possible but might sometimes use stack-specific language for clarity.
+
+For any given rollup there are five main concerns inherent to an integration
 with external DA:
 
 1. **Dispersal.** The rollup batcher must write transaction batches to the DA
 layer, wait for confirmation, and write the resulting DA certificate to the
-rollup inbox.
-2. **Certificate Verification.** Either the rollup inbox contract
+[rollup inbox][glossary-rollup-inbox].
+1. **Certificate Verification.** Either the rollup inbox contract
 or the rollup OS must verify that DA certificate is valid, i.e. that enough
 operators have certified the blob available, before reading the DA cert's data
 from the DA layer. This ensures that a transaction batch referenced by an
 invalid certificate is not executed.
-3. **Retrieval.** Rollup full nodes must retrieve EigenDA blobs as part of the
+   1. **Certificate Punctuality (Timing) Verification.** The certificate must be posted to the batcher inbox within some punctuality window. EigenDA blobs are only available to download for 2 weeks, so this check is necessary to prevent malicious sequencers from posting certificates right before the blob gets deleted.
+1. **Retrieval.** Rollup full nodes must retrieve EigenDA blobs as part of the
 L2 derivation/challenge process. Otherwise they cannot keep up with the state of
 the L2.
-4. **Data Verification.** The rollup's fraud arbitration protocol must be
-capable of verifying that the EigenDA blob data used to generate a state root
+1. **Blob Commitment Verification.** The rollup's fraud arbitration protocol must be
+capable of verifying that every EigenDA blob used to generate a state root
 matches the KZG commitment provided in the EigenDA cert posted to the rollup
 inbox. In doing this verification, the chain ensures that the transaction data
 used to generate the rollup's state root was not manipulated by the
 sequencer/proposer.
 
-Each strategy used to integrate EigenDA can be defined by how these four concerns
-are handled.
+A fully secure integration requires doing the 3 verification checks.
+
+|                                 | Dispersal | Retrieval | Cert Verification | Blob Verification | Timing Verification |
+| ------------------------------- | --------- | --------- | ----------------- | ----------------- | ------------------- |
+| Trusted | x         | x         |                   |                   |                     |
```

Comment:
Nit: whitespace isn't aligned in this markdown table, which makes it difficult to read in raw form

Reply:
I think its just the github view..? The table is getting formatted by my markdown extension. Looks like this on my IDE:
![image](https://github.com/user-attachments/assets/90fd9fe3-822f-4bde-a115-3a2f131dcf79)

Reply:
I looks like whatever extension changes you made fixed the problem, in [this commit](https://github.com/Layr-Labs/eigenda-docs/pull/72/commits/bcd8c475cf49d8b787fed41cce88c687245d151e). Looks good now